janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server.yaml

```diff
@@ -17,7 +17,7 @@ port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
 graphs: {
-  graph: conf/gremlin-server/janusgraph-cql-es-server.properties
+  graph: conf/gremlin-server/janusgraph-inmemory.properties
```

Comment:
Why not use conf/janusgraph-inmemory.properties?

Reply:
Changed to `conf/janusgraph-inmemory.properties`